### PR TITLE
make acceptableContentTypes thread safe.

### PR DESCRIPTION
**Pull request by Lobster-King**

### CODE REVIEW

**Comment by Lobster-King** [comment]
What's the latest branch？

File: AFNetworking/AFURLResponseSerialization.m [hunk 1]
```diff
@@ -202,6 +210,24 @@ - (instancetype)copyWithZone:(NSZone *)zone {
...
+- (NSSet <NSString *> *)acceptableContentTypes {
+    NSSet <NSString *> __block *value;
+    dispatch_sync(self.requestContentTypesModificationQueue, ^{
+        value = _acceptableContentTypes;
+    });
+    return value;
+}
+
+- (void)setAcceptableContentTypes:(NSSet<NSString *> *)acceptableContentTypes {
+    dispatch_barrier_sync(self.requestContentTypesModificationQueue, ^{
```
Comment (Legonaftik) [inline AFNetworking/AFURLResponseSerialization.m:224]: It's clear why you use `_sync` for reading but it seems like `_sync` for writing with barrier is redundant. 
What about using `dispatch_barrier_async` here instead?

**Review by Legonaftik — APPROVED** [review]

File: AFNetworking/AFURLResponseSerialization.m (hunk 1 above)
Comment (Lobster-King) [reply AFNetworking/AFURLResponseSerialization.m:224]: yes,we can use dispatch_barrier_async instead .I'll push a patch for this commit .thx for your review !

**Comment by Lobster-King** [comment]
https://github.com/AFNetworking/AFNetworking/pull/4645 @Legonaftik new pr

**Pull request closed without merging**